### DISCUSSION TITLE
Path aliases replacement on transform

### DISCUSSION THREAD

**Discussion by jonathansayag** [swc-project/swc#7779] [Q&A]
Hi everyone !

I have an issue regarding the usage of path aliases with SWC. 

**Context**
In my project, I am importing some libs from the folder node_modules like `import('rematrix')`. 

I am trying to use the JS api of SWC & transformSync() to transform this JS into that for example: 
```
import('../../external-modules/rematrix/index.js')
```

**Note** that I added the path alias to the transformSync option. 

But nothing is working, I still have `import('rematrix')` in the output of the transformation. 

Thanks for your help !

**Comment by kdy1** [comment]
Use `jsc.paths`

**Reply by jonathansayag** [reply]
Hi,

That's what I already tried before, using this kind of config:
```
{
  "jsc": {
    ...
    "baseUrl": "./",
    "paths": {
      "rematrix": ["external-modules/rematrix.js"]
    },
    ...
  }
}
```

But it still keeping `import('rematrix')`. 

Does the file has to exists before running the transform function ?
